plugins/rest/rest_auth.go

```diff
@@ -39,30 +39,12 @@ func defaultRoundTripperClient(t *tls.Config, timeout int64) *http.Client {
 	// Ensure we use a http.Transport with proper settings: the zero values are not
 	// a good choice, as they cause leaking connections:
 	// https://github.com/golang/go/issues/19620
-	//
-	// Also, there's no simple way to copy the values from http.DefaultTransport,
-	// see https://github.com/golang/go/issues/26013. Hence, we copy the settings
-	// used in the golang sources,
-	// https://github.com/golang/go/blob/5fae09b7386de26db59a1184f62fc7b22ec7667b/src/net/http/transport.go#L42-L53
-	//   Copyright 2011 The Go Authors. All rights reserved.
-	//   Use of this source code is governed by a BSD-style
-	//   license that can be found in the LICENSE file.
-	var tr http.RoundTripper = &http.Transport{
-		Proxy: http.ProxyFromEnvironment,
-		DialContext: (&net.Dialer{
-			Timeout:   30 * time.Second,
-			KeepAlive: 30 * time.Second,
-			DualStack: true,
-		}).DialContext,
-		MaxIdleConns:          100,
-		IdleConnTimeout:       90 * time.Second,
-		TLSHandshakeTimeout:   10 * time.Second,
-		ExpectContinueTimeout: 1 * time.Second,
-		ResponseHeaderTimeout: time.Duration(timeout) * time.Second,
-		TLSClientConfig:       t,
-	}
 
 	// copy, we don't want to alter the default client's Transport
```

Comment:
[nit] I believe this comment still makes a little more sense [above line 48](
https://github.com/open-policy-agent/opa/pull/2779/files#diff-9342537850bc2653ee3e23ca0902c00a247ebf2cc504fa03f385ef9a884c7b85R48), maybe?

Reply:
You’re probably right. I’ll fix it in my next PR, unless you beat me to it 😊